### PR TITLE
Feature/add view action icon

### PR DESCRIPTION
**Pull request by severinoAmmirati**
Added custom icon to view action column

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.2.1] - 2023-02-22
```
Comment (000-ooo-000) [inline CHANGELOG.md:10]: Remove this, it is not needed, it'll be set automatically on release

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Added
+
+-   the prop `icon` to the `DataTablePredefinedActionLink` interface to specify the view action column icon. Possible values are all `IconProp`. The default value is the eye icon `faEye`
+
```
Comment (000-ooo-000) [inline CHANGELOG.md:15]: Probably not building due to having an extra new line

**Review by 000-ooo-000 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (severinoAmmirati) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 1 above)
Comment (severinoAmmirati) [reply CHANGELOG.md]: Version removed

**Review by 000-ooo-000 — APPROVED** [review]

**Review by sineao — APPROVED** [review]

**Review by neotob — APPROVED** [review]

**Pull request merged**